examples/comments/README.md

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
Please fill this out :)

Reply:
sorry, I think this was from an older commit it does have content.